test/irb/test_context.rb

```diff
@@ -145,6 +145,48 @@ def test_output_to_pipe
       assert_equal "=> 1\n", out
     end
 
+    def test_eval_object_without_inspect_method
+      verbose, $VERBOSE = $VERBOSE, nil
+      all_assertions do |all|
+        IRB::Inspector::INSPECTORS.invert.each_value do |mode|
```

Comment:
I think this is equivalent

```suggestion
        IRB::Inspector::INSPECTORS.each_key do |mode|
```